ledger/block/src/transaction/deployment/mod.rs

```diff
@@ -113,15 +113,10 @@ impl<N: Network> Deployment<N> {
     }
 
     /// Returns the number of program functions in the deployment.
-    pub fn len(&self) -> usize {
+    pub fn num_functions(&self) -> usize {
```

Comment:
For consistency, can you also change the equivalent function for the execution? https://github.com/ProvableHQ/snarkVM/blob/staging/ledger/block/src/transaction/execution/mod.rs#L120

Reply:
I would prefer to keep these two concepts unrelated. If Deployment requires is_empty, we can rediscuss bringing it back to len.